test/utils.js

```diff
@@ -7,7 +7,11 @@ const path = require('path');
 const WebSocket = require('ws');
 
 beforeEach(function() {
```

Comment:
This fails on Windows, and is fatal when it does. To allow the tests to continue executing past `hmr`, this is catching and logging the error. This can / should be removed once the tests are fixed on Windows.